forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabMergeRequest.java

```diff
@@ -830,8 +830,14 @@ public URI headUrl() {
     @Override
     public Diff diff() {
         var changes = request.get("changes").param("access_raw_diffs", "true").execute();
+        boolean diffComplete;
+        if (changes.get("overflow").asBoolean()) {
```

Comment:
During my experiment, I found we can't trust the "overflow" field returned by GitLab.